test/test-conn.py

```diff
@@ -168,11 +168,12 @@ def send_knot_msg_auth(sock, uuid, token):
     return sock.send(msg)
 
```

Comment:
int the commit 1c41ee1fcb00d13b9a44b74766d418242d6f4756:

Send schema end if it is the last one in test-conn

Reply:
done